hadoop-hdds/docs/content/feature/OM-HA.md

```diff
@@ -125,7 +125,23 @@ ozone om [global options (optional)] --bootstrap --force
 
 Note that using the _force_ option during bootstrap could crash the OM process if it does not have updated configurations.
 
+## Automatic Snapshot Installation for Stale Ozone Managers
+
+Sometimes an OM follower node may be offline or fall so far behind the leader OM's log that it cannot catch up by replaying individual log entries.  The OM HA implementation includes an automatic snapshot installation and recovery process for such cases.
+
+How it works:
+
+1. Leader determines that the follower is too far behind.
+2. Leader notifies the follower to catch up via snapshot.
+3. The follower downloads and installs the latest snapshot from the leader.
+4. After installing the snapshot, the follower OM resumes normal operation and log replication from the new state.
+
+This logic is implemented in the [`OzoneManagerStateMachine.notifyInstallSnapshotFromLeader()` method](https://github.com/apache/ozone/blob/931bc2d8a9e8e8595bb49034c03c14e2b15be865/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java#L521-L541).
```

Comment:
Let's point to a particular release:
```md
This logic is implemented in the `OzoneManagerStateMachine.notifyInstallSnapshotFromLeader()`;
see the [code](https://github.com/apache/ozone/blob/ozone-2.0.0/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java#L520-L531)
in Release 2.0.0.
```

It would be great if there is a way to specify a base URL variable something like below:
```
BASE_URL=https://github.com/apache/ozone/blob/ozone-2.0.0
```
Then the link becomes
```md
[code](${BASE_URL}/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java#L520-L531)
```